chapters/synchronization.txt

```diff
@@ -549,8 +549,9 @@ guaranteed:
   * ename:VK_PIPELINE_STAGE_HOST_BIT
 
 ifdef::VK_EXT_conditional_rendering[]
-For conditional rendering operations, the pipeline stage where predicate
-read happens has no particular order relative to other stages.
+The conditional rendering stage is formally part of both the graphics, and
```

Comment:
I see that #742 wasn't squashed to a single commit when I cherry-picked it into our internal gitlab, which is why this didn't make it in. That said, we think it would be clearer to actually list this bit as part of both the graphics and compute stage lists, probably with a reference to the "unspecified order" comment here rather than repeating it in both lists.

Reply:
Alternatively, it can be another pseudo-pipeline, like device-generated commands pipeline... seems very similar case.

BTW shouldn't `VK_PIPELINE_STAGE_COMMAND_PROCESS_BIT_NVX` be allowed in `VkSubpassDependency` too?

Reply:
We considered calling it a pseudo stage initially (merge request 2676) but later decided against it. I am fine with this change.

Reply:
@oddhack I am not sure how to practically wordsmith that. The graphics and compute pipelines say "the following stages occur in this order:". I am not sure how to clearly list this bit there as unordered.

Some improvement could be to move this paragraph below COMPUTE, so it is at least closer to those related pipelines.

Reply:
Yes that would be a slight improvement. I dont think its necessary to add the bit to either graphics or compute pipeline stages in the paragraph above.

Reply:
I made it so then.